src/common/sv2_messages.h

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2023-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_COMMON_SV2_MESSAGES_H
+#define BITCOIN_COMMON_SV2_MESSAGES_H
+
+#include <common/net_message.h>
+#include <span.h>
+#include <streams.h>
+#include <string>
+#include <util/check.h>
+
+namespace node {
```

Comment:
639be6df22800d2e2a170c1af74d51951597834c: this `namespace` is outdated, since this isn't part of the `node` anymore.